### DISCUSSION TITLE
Playtime 2 Guitar Looping Idea

### DISCUSSION THREAD

**Discussion by BradleyBemis** [helgoboss/helgobox#1284] [Q&A] [answered]
Hi all,

The Idea:
I'm using a looping technique in REAPER with Playtime 2 where I have an "IN" track set up for my guitar input. This track is armed and set to monitor  "ON" and has my preferred guitar VST effects applied (McRocklin Suite by Polychrome DSP). The audio from the "IN" track is routed to multiple other tracks that are armed for recording but set to monitoring "OFF." These tracks receive the audio from the "IN" track, allowing them to record sound without monitoring, while playback is managed through Playtime 2.

How It Works:
The "IN" Track: This track is always active and ready for input. I can dynamically adjust effects during performance, and any changes will be recorded on the armed tracks.
Recording and Looping: To record a new loop, I simply tap an empty slot in Playtime 2. The audio is recorded on one of the armed tracks. If I need to add a new layer, I choose another slot without needing to re-arm tracks.
Layering and Playback: This technique allows for seamless layering and switching between loops without interrupting the input signal. The "IN" track remains dedicated to live input, and loop playback doesn’t override it.

Additional Setup:
I'm using the McRocklin Suite for guitar processing, which lets me control presets and effects using my Morningstar MC8 MIDI controller. This setup functions similarly to a traditional pedalboard, offering hands-free operation for switching tones and effects during looping.

Current Issue:
So far, I have been successful in getting the signal to record and playback; however, it only plays back the dry guitar signal without FX. I'm looking for a way to ensure that the recorded audio plays back with the effects applied.  

Any thoughts?  Thank you!

**Comment by RibbleFripFnibbulator** [comment, marked as answer]
Hey Bradley, it sounds like you need to add an Audio Loopback Output from your "IN" track and then in Playtime set the recording track to the equivalent loopback input. That way you'll print the effects into the loops (as long as the loopback output is post-fx which it is by default). As far as I understand, using loopback channels is the only way to do this currently, but eventually we'll be able to record directly from any track output. You also have to set the number of loopback channels in Reaper Preferences -> Audio, as it defaults to zero.

**Reply by BradleyBemis** [reply]
Hi Ribble,

Thank you for your help!

You have a solid grasp of what I’m trying to accomplish. Unfortunately, I tried the approach you suggested, but I still couldn’t get the audio to imprint using the loopback channel. I experimented with a few different configurations, but I’m still stumped for now.

**Reply by RibbleFripFnibbulator** [reply]
If you want to share a Reaper project I'd be happy to take a look.

**Reply by BradleyBemis** [reply]
Hi Ribble,

Thanks for following up!

Yesterday, I managed to get the loopback audio to successfully print on the dummy tracks with effects! I found a helpful video from the Reaper Blog that showed me how to add Loopback as an Audio Hardware Output on my guitar "IN" channel, which turned out to be my missing step.

However, I’m now facing an issue where the dummy tracks are boosting my guitar signal, even with monitoring set to "off." I’ve confirmed this by noticing a significant drop in guitar level when I mute those tracks. Is there a way to prevent this from happening?

Thanks for your help!

![image](https://github.com/user-attachments/assets/c21416b0-14c9-4373-925e-e3da832dcc30)

![image](https://github.com/user-attachments/assets/80514a7c-316c-4b43-8b83-be17d0a477ac)

![image](https://github.com/user-attachments/assets/bbe75c00-3e3a-4fbb-a8e3-d047c97b35b5)

![image](https://github.com/user-attachments/assets/cf071350-847a-486b-9327-c5a8e7ff129e)

**Reply by RibbleFripFnibbulator** [reply]
Hi Bradley,

You don't need those sends from Guitar to the dummy tracks anymore. Once the Dummy tracks' input are set to the loopback channel outputted from Guitar, that's all you need. Those additional sends are "double-sending" on top of the loopback so that's messing with the levels. Just kill those and you should be good.

**Reply by BradleyBemis** [reply]
Hi Ribble,

You solved it!

Thanks so much, I really appreciate your help.
